src/pages/redirect-page.tsx

```diff
@@ -1,97 +1,148 @@
-import React, { useCallback, useContext, useEffect, useMemo, useState, type ReactElement } from 'react'
-import Header from '../components/Header.jsx'
-import { ConfigProvider } from '../context/config-context.jsx'
-import { ServiceWorkerContext, ServiceWorkerProvider } from '../context/service-worker-context.jsx'
-import { setConfig, type ConfigDb } from '../lib/config-db.js'
-import { getSubdomainParts } from '../lib/get-subdomain-parts.js'
-import { isConfigPage } from '../lib/is-config-page.js'
-import { getUiComponentLogger, uiLogger } from '../lib/logger.js'
-import { tellSwToReloadConfig } from '../lib/sw-comms.js'
-import { translateIpfsRedirectUrl } from '../lib/translate-ipfs-redirect-url.js'
-import './default-page-styles.css'
-import LoadingPage from './loading.jsx'
-import './loading.css'
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+  type ReactElement,
+} from "react";
+import Header from "../components/Header.jsx";
+import { ConfigProvider } from "../context/config-context.jsx";
+import {
+  ServiceWorkerContext,
+  ServiceWorkerProvider,
+} from "../context/service-worker-context.jsx";
+import { setConfig, type ConfigDb } from "../lib/config-db.js";
+import { getSubdomainParts } from "../lib/get-subdomain-parts.js";
+import { isConfigPage } from "../lib/is-config-page.js";
+import { getUiComponentLogger, uiLogger } from "../lib/logger.js";
+import { tellSwToReloadConfig } from "../lib/sw-comms.js";
+import { translateIpfsRedirectUrl } from "../lib/translate-ipfs-redirect-url.js";
+import "./default-page-styles.css";
+import LoadingPage from "./loading.jsx";
+import "./loading.css";
 
-const uiComponentLogger = getUiComponentLogger('redirect-page')
-const log = uiLogger.forComponent('redirect-page')
+const uiComponentLogger = getUiComponentLogger("redirect-page");
+const log = uiLogger.forComponent("redirect-page");
```

Comment:
@Winter-Soren it looks like you have some editor auto-format-and-save configured that isn't using our linting rules. Can you disable that and update the PR? there are a lot of changes here that aren't related to the actual fix to #530